starknet_py/net/networks.py

```diff
@@ -3,19 +3,16 @@
 from starknet_py.constants import FEE_CONTRACT_ADDRESS
 
 MAINNET = "mainnet"
-GOERLI = "goerli"
 SEPOLIA_TESTNET = "sepolia_testnet"
```

Comment:
I know this was already here, but I think this should not be `sepolia_testnet` but rather `sepolia`